<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>611</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>591</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>Main Window</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>58</y>
        <width>571</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item row="0" column="2">
        <widget class="QToolButton" name="mw_background_color_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QToolButton" name="mw_selected_color_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="mw_font_color_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fontSizeLabel">
         <property name="text">
          <string>Font size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="mw_font_color"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="mw_font_size"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Menu selected item background color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Font color</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="mw_selected_color"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Background color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="mw_background_color">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFontComboBox" name="mw_font">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentFont">
          <font>
           <family>Segoe UI</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>591</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>Text Editor</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>571</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="2" column="2">
        <widget class="QToolButton" name="te_font_color_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="te_background_color_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="te_background_color"/>
       </item>
       <item row="1" column="1">
        <widget class="QFontComboBox" name="te_font">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Background color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="te_font_size"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fontSizeLabel_2">
         <property name="text">
          <string>Font size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="te_font_color"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Font color</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>591</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>Syntax Highlight</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>59</y>
        <width>571</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="1">
        <widget class="QLineEdit" name="string"/>
       </item>
       <item row="7" column="2">
        <widget class="QToolButton" name="self_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Keywords</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="keywords"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="keywords_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="self_l"/>
       </item>
       <item row="4" column="2">
        <widget class="QToolButton" name="string_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Multililine string</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Operators</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="operators"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>def and class</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="defclass"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Braces</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="operators_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="multiline_string"/>
       </item>
       <item row="5" column="2">
        <widget class="QToolButton" name="multiline_string_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Comments</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Self</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="braces"/>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="braces_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="defclass_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>String</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QToolButton" name="comments_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="comments"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Numbers</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="numbers"/>
       </item>
       <item row="8" column="2">
        <widget class="QToolButton" name="numbers_toolbtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_new_preset_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save as new preset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_preset_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load preset</string>
   </property>
  </widget>
  <widget class="QComboBox" name="page_combo_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Main Window</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Text Editor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Syntax Highlight</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>490</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save and apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Preset:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="current_preset_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete preset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
